swatch-producer-aws/build.gradle

```diff
@@ -23,7 +23,9 @@ dependencies {
     implementation 'io.quarkus:quarkus-logging-json'
     implementation 'io.quarkus:quarkus-micrometer-registry-prometheus'
     implementation 'io.quarkus:quarkus-openshift'
-    implementation 'io.quarkus:quarkus-quartz'
+    implementation 'io.quarkus:quarkus-rest-client-jackson'
+    implementation 'io.quarkus:quarkus-rest-client'
+    implementation 'io.quarkus:quarkus-resteasy-jsonb'
```

Comment:
I was thinking we just get these transitively as needed from included client jar libs. Any reason not to do that?

Reply:
I think just a whoops during rebasing